docs/spec/resource-scopes.md

```diff
@@ -83,21 +106,15 @@ This feature will be limited to the same scoping constraints that exist within A
 ## Compatibility with upcoming features
 
 ### Defaults
```

Comment:
Maybe we just cut defaults for now? It's a pretty loosely formed idea atm and I don't think we have a spec/issue with code samples yet

Reply:
Sounds good to me!